verification/cbmc/proofs/aws_cryptosdk_hdr_size/cbmc-batch.yaml

```diff
@@ -0,0 +1,8 @@
+build_memory: 32000
+cbmcflags: "--unwind;1;--unwindset;aws_cryptosdk_edk_list_elements_are_bounded.0:3,aws_cryptosdk_edk_list_elements_are_valid.0:3,aws_cryptosdk_hdr_size.0:3,ensure_cryptosdk_edk_list_has_allocated_list_elements.0:3,memcmp.0:1029;--flush;--object-bits;8;--external-sat-solver;kissat;--malloc-may-fail;--malloc-fail-null;--bounds-check;--conversion-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--pointer-primitive-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwinding-assertions;--malloc-may-fail;--malloc-fail-null"
+coverage_memory: 32000
+expected: SUCCESSFUL
+goto: /Users/tegab/clean/aws-encryption-sdk-c/verification/cbmc/proofs/aws_cryptosdk_hdr_size/gotos/aws_cryptosdk_hdr_size_harness.c.goto
+jobos: ubuntu16
+property_memory: 32000
+report_memory: 32000
```

Comment:
I don't think we need this `cbmc-batch.yaml` file anymore, you can replace it with the standard one.
```Makefile
:
  This file marks this directory as containing a CBMC proof. This file
  is automatically clobbered in CI and replaced with parameters for
  running the proof.
```

Reply:
Updated!